app/posts/2021-08-07-ab-test-job-alert-validation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'A/B test: requiring users to be more specific when creating a job alert'
+description: Increasing job alert quality doesn’t need to sacrifice traffic volume.
+date: 2021-08-07
+related:
+  items:
+  - text: Performance review of job alerts feature (November 2020)
+    href: https://docs.google.com/presentation/d/13sa9poO2PSCgPT1D2GmSgcYsjRD-v4P7h9V7xjDKFL0/edit?usp=sharing
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+Initial analysis of the performance of our job alerts feature found that when users set up job alerts with less specific criteria - i.e. not specifying a location or what kind of job they were looking for, or not specifying anything at all - when they later received emails triggered by that alert, they were less likely to click on a vacancy included in the email. We assumed that this was because the vacancies in the email were likely to be less relevant to them if they had been less specific about what they were looking for.
```

Comment:
Is the email triggered by the alert, or is the email itself the alert?

Reply:
One for Steven if you want to debate it @bencmitchell !